src/services/database/models/user.js

```diff
@@ -3,76 +3,143 @@ const { Schema, model } = require('mongoose');
 // Create the Mongoose schema
 const UserSchema = new Schema(
   {
-    firstname: {
+    firstName: {
       type: String,
       required: true,
       trim: true,
     },
-    lastname: {
+    lastName: {
       type: String,
-      required: true,
       trim: true,
     },
     username: {
       type: String,
       required: true,
       unique: true,
       trim: true,
+      minLength: 3,
```

Comment:
Let's see if we can make it have a naming convention like first letter has to be capital and have to include a special char